flake.nix

```diff
@@ -249,6 +270,29 @@
             mkdir $out # success
           '';
 
+          checks.age-plugin = pkgs.runCommand "age-plugin" { } ''
+            set -euo pipefail
+            cp -r '${./.}/example/.' "$TMPDIR"
+            cd "$TMPDIR/plugin-example"
+
+            PATH=$PATH:${exampleRagePlugin}/bin
+
+            # Encrypt with ragenix
+            echo wurzelpfropf | ${pkgs.ragenix}/bin/ragenix --editor - --edit unencrypted.age
+
+            # Decrypt with rage
+            decrypted="$(${pkgs.rage}/bin/rage -i '${./example/keys/example_plugin_key.txt}' -d unencrypted.age)"
```

Comment:
As bonus, you could also do the same test with `pkgs.age`.

Reply:
That unfortunately breaks, as Age does not support plugins yet:
https://github.com/str4d/age-plugin-yubikey/issues/43#issuecomment-986013096

Reply:
Ah, got it.